ui/pages/confirmations/confirm/index.scss

```diff
@@ -4,6 +4,6 @@
   box-shadow: var(--shadow-size-lg) var(--color-shadow-default);
 
   @media screen and (min-width: design-system.$break-small) {
-    max-width: 62%;
+    max-width: 360px;
   }
```

Comment:
This value 360px is taken from figma designs https://www.figma.com/design/H52Ob2wDWsPY4g9DeYuCLk/Confirmation-redesign-V6?node-id=3433-27766&node-type=FRAME&t=N7Ij9OuBbvoIlqrJ-0